### DISCUSSION TITLE
Prevent reruning Queries after mutation

### DISCUSSION THREAD

**Discussion by T0miii** [urql-graphql/urql#2129] [Q&A]
Hello,

so i know about the fact that after useMutation the __typneame is used to rerun all queries that have the same type.
Is there a wait to disable this feature, or just enable it for specific queries?

**Comment by kitten** [comment]
I'm not sure of your use-case, but generally more advanced use-cases like this is what we're aiming to cover with the normalised cache. That does mean that you're jumping to the other end of the "caching spectrum", since you'll have automatic normalised updates but it's "never invalidating by default" rather than "always invalidating by default".

But if you provide a short gist of what you're trying to achieve and what you've got I can probably give you some more concrete suggestions and options :v:

**Reply by adiazarya100** [reply]
Hello,
We would like to prevent the re-run of the queries since we have multi queries in the same component the component fires all of them which makes our app to re-render each part again.
What do you suggest?

for example, I have a table populates using a computed variable (Vue 3) 
after editing one item in the table (for example status) the component fires a request to fetch all the items again. ~200 items. and re-render the table.

This is my code:

```
const { data, error, fetching } = useGetUsers();
const tableData = computed(() => data.value.users);
```

**Reply by kitten** [reply]
We've got the suggestion right here 😅 Use Graphcache, which is our normalised cache https://formidable.com/open-source/urql/docs/graphcache/

**Reply by T0miii** [reply]
This means, there is now way to turn of this feature without using an additional plugin?

The use case again would be:

Dashboard with -> items
Edit Item useMutation to persist changes.
Dashboard gets refetched cause item got updated. <--- Would like to prevent this refetch
